sklearn/linear_model/_ridge.py

```diff
@@ -218,7 +220,7 @@ def _solve_cholesky_kernel(K, y, alpha, sample_weight=None, copy=False):
 
 
 def _solve_svd(X, y, alpha):
-    U, s, Vt = linalg.svd(X, full_matrices=False)
+    U, s, Vt = linalg.svd(X, full_matrices=False, check_finite=False)
```

Comment:
But I prefer to not have `check_finite=False` here because it is not obvious that X is finite. One needs to go through the code to check that the caller did the check.